contracts/Account.sol

```diff
@@ -2,34 +2,19 @@
 
 pragma solidity ^0.8.0;
 
-import {SignatureRSV, EthereumUtils} from "@oasisprotocol/sapphire-contracts/contracts/EthereumUtils.sol";
-import {EIP155Signer} from "@oasisprotocol/sapphire-contracts/contracts/EIP155Signer.sol";
-import {CloneFactory} from "./lib/CloneFactory.sol";
-
-contract AccountFactory is CloneFactory {
-    Account private account;
-
-    constructor () {
-        account = new Account();
-    }
-
-    function clone (address starterOwner)
-        public
-        returns (Account acct)
-    {
-        acct = Account(createClone(address(account)));
-        acct.init(starterOwner);
-    }
+struct Wallet {
+    address keypairAddress;
+    string title;
 }
 
-contract Account {
-    bool private _initialized;
+abstract contract Account {
+    bool internal _initialized;
 
-    mapping(address => bool) private _controllers;
+    mapping(address => bool) internal _controllers;
```

Comment:
Possibly controllers could be replaced by openzeppelins implementation for roles?